### PR TITLE
Test-running improvements

### PR DESCRIPTION
**Pull request by hobarrera**
Minor improvements to how tests are ran, as well as running tests with a negative UTC offset which might help catch bugs earlier in future.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -9,7 +10,18 @@ deps =
 commands =
 	py.test {posargs}
 
+[testenv:negtz]
+passenv = LANG
+setenv =
+    TZ = America/Argentina/Buenos_Aires
+deps =
```
Comment (untitaker) [inline tox.ini]: I'm not really sure, but I suspect that these params could be inherited from `[testenv]`.

Comment (hobarrera) [inline tox.ini]: Yup, you're right: https://testrun.org/tox/latest/config.html#substitution-for-values-from-other-sections

I'll look into it later today.

Comment (hobarrera) [inline tox.ini]: These seems to be a bug in `tox` that prevents us from using this 100%:

`commands = {[testenv]commands}`

Does not substitute `{posargs}` in `commands` before the inheritance :(

Comment (untitaker) [inline tox.ini]: I mean this part:

    Defaults for each setting in this section are looked up in the:

    [testenv]
    ...

    testenvironment default section.

Comment (hobarrera) [inline tox.ini]: Duh! Thanks. Fixed.

**Comment by untitaker** [comment]
LGTM, thanks!

**Pull request merged**